.gitbook/guide/installation.md

```diff
@@ -1,39 +1,29 @@
 # Installation
 
-This guide will explain how to install the `panacead` and `panaceacli` entrypoints onto your system.
+This guide will explain how to install the `panacead` entrypoints onto your system.
 
 ## Install Go
 
 [Go 1.15+](https://golang.org/doc/install) is required.
 
-## Install the `panacead` and `panaceacli`
+## Install the `panacead`
 
 Install the latest version of Panacea Core.
 
 ```bash
 git clone https://github.com/medibloc/panacea-core
 cd panacea-core
-git checkout v1.3.3
+git checkout v2.0.0-alpha.1
```

Comment:
I'm thinking about  whether to change this version to the official version(v2.0.0)

Reply:
I think that `v2.0.0-alpha.1` is good. When we release next versions, we should keep in mind that this doc also should be updated.